polynote-frontend/polynote/ui/util/tags.ts

```diff
@@ -138,6 +151,13 @@ export function textbox(classes: string[], placeholder: string, value: string =
     if (value) {
         input.value = value;
     }
+    input.addEventListener('keydown', (evt: KeyboardEvent) => {
```

Comment:
TIL there are `Cancel` and `Accept` keys haha (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) 

So those are fired when you use the keyboard to select a Cancel or Accept button?

Reply:
Yeah I was just looking for `Escape` and `Enter` on that list to make sure I had the right stringlies... and since `Cancel` and `Accept` were on the list I figured I should listen to them? /shrug